Simplified/NYPLSettingsAccountDetailViewController.m

```diff
@@ -1520,7 +1520,9 @@ - (UITableViewCell *)tableView:(__attribute__((unused)) UITableView *)tableView
           }];
           [self.barcodeImageView autoPinEdgeToSuperviewEdge:ALEdgeTop withInset:sConstantSpacing];
           [self.barcodeImageLabel autoAlignAxisToSuperviewAxis:ALAxisVertical];
-          [self.barcodeImageLabel autoPinEdgeToSuperviewEdge:ALEdgeBottom withInset:10.0];
+          [NSLayoutConstraint autoSetPriority:UILayoutPriorityDefaultHigh forConstraints:^{
```

Comment:
did you verify this on a physical device too? If I remember correctly, on a physical device there was a glitchy line being displayed after hiding the barcode.

Reply:
It works fine on a physical iPhone xs (iOS 13.7)